test/test_browser.py

```diff
@@ -4643,14 +4661,21 @@ def test_webgl_simple_enable_extensions(self):
                '-sGL_SUPPORT_SIMPLE_ENABLE_EXTENSIONS=' + str(simple_enable_extensions)]
         self.btest_exit('webgl2_simple_enable_extensions.c', args=cmd)
 
+  @parameterized({
+    '': ([],),
+    'closure': (['-sASSERTIONS', '--closure=1'],),
+    'main_module': (['-sMAIN_MODULE=1'],),
+  })
   @requires_graphics_hardware
-  def test_webgpu_basic_rendering(self):
-    for args in [[], ['-sASSERTIONS', '--closure=1'], ['-sMAIN_MODULE=1']]:
-      self.btest_exit('webgpu_basic_rendering.cpp', args=['-sUSE_WEBGPU'] + args)
+  def test_webgpu_basic_rendering(self, args):
+    self.btest_exit('webgpu_basic_rendering.cpp', args=['-sUSE_WEBGPU'] + args)
 
-  def test_webgpu_get_device(self):
-    for args in [['-sASSERTIONS', '--closure=1']]:
-      self.btest_exit('webgpu_get_device.cpp', args=['-sUSE_WEBGPU'] + args)
+  @parameterized({
+    '': ([],),
```

Comment:
It looks like this is a new mode being tested, as the old code had a loop but just on a single value of both assertions+closure? Is this new mode worth testing?

Reply:
Oh .. I see we can just remove the loop.  I will do that.